sass/_OmniboxResultList.scss

```diff
@@ -2,5 +2,12 @@
 	&.coveo-omnibox-selectable {
 	  padding: 10px 10px 10px 36px;
 	  margin: 0;
+		box-sizing: border-box;
+	}
+	&.coveo-omnibox-result-list-element.coveo-omnibox-selectable.coveo-omnibox-selected {
```

Comment:
Yes, this selector needs to be this precise to override the style from the field suggestion / analytics suggestion.